src/components/Page/Page.tsx

```diff
@@ -81,8 +87,7 @@ export class Page extends React.PureComponent<ComposedProps, never> {
     );
 
     const headerMarkup =
-      this.props.polaris.appBridge ||
-      this.hasHeaderContent() === false ? null : (
+      this.delegateToAppbridge || this.hasHeaderContent() === false ? null : (
```

Comment:
I think I had some feedback on this previously, can we add brackets to make the logic here explicit at a glance? 
```
(this.delegateToAppBridge ||  this.delegateToAppbridge || this.hasHeaderContent() === false) ? null : ...
```
vs
```this. delegateToAppbridge || (this.hasHeaderContent() === false ? null : ...)`

Reply:
We could do this, but it will throw some linting errors for "uneeded" brackets, so if we wanted to go down this route we'd have to add a comment ignoring the linting rule first, which I don't think is a great option.

Reply:
For styling and lint rules, we largely use the Shopify config. You can open an issue here https://github.com/Shopify/eslint-plugin-shopify or if it is a project-specific setting, open an issue here.